Assets/UnityVoiceProcessor/Demo/Recorder.cs

```diff
@@ -0,0 +1,97 @@
+﻿//
+// Copyright 2023 Picovoice Inc.
+//
+// You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+// file accompanying this source.
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+//
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Pv.Unity
+{
+    class Recorder

```

Comment:
I know it's pedantic, but calling this 'Recorder' isn't very accurate. It's more of a WavFileWriter or something